src/main/scala/io/sdkman/changelogs/java/BellSoftLibericaMigrations.scala

```diff
@@ -361,4 +361,143 @@ class BellSoftLibericaMigrations {
       .insert()
   }
 
+  @ChangeSet(
+    order = "0029",
+    id = "0029-add_bellsoft_8_0_272",
+    author = "poad"
+  )
+  def migrate0029(implicit db: MongoDatabase): Unit = {
+
+    Map(
+      LinuxARM64 -> "bellsoft-jdk8u272+10-linux-aarch64.tar.gz",
+      Linux32    -> "bellsoft-jdk8u272+10-linux-i586.tar.gz",
```

Comment:
@marc0der are we providing for Linux32? Just asking because have seen this before.